README.md

```diff
@@ -257,7 +257,14 @@ types are hashable as well, so you can construct stranger composite types
       StringIntegerMap(brian => 28, ian => 15, robey => 5000) => 0.2,
       StringIntegerMap(brian => 30, ian => 21, robey => 35) => 0.9)
 
+#### Choice Types ####
```

Comment:
This is at the same header level as List and Map under 'Container types' which does not jive with the section's intro text nor the 'The type system' sentence above that which calls out Choice as different from the container types.